src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,91 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Random;
+import java.util.StringJoiner;
+
+public class RandomCharsTable {
```

Comment:
Works good, but maybe you can simplify for better readability

Reply:
done